packages/rocketchat-ui-message/client/messageBox.js

```diff
@@ -153,7 +158,12 @@ Template.messageBox.helpers({
 		});
 
 		return columns;
-	},
+	}
+};
+
+Template.messageBox__actions.helpers({...methods});
```

Comment:
you can pass `methods` variable directly, don't need to spread it into another object

Reply:
I know, but we are suggesting that we can put more functions inside.

Reply:
I`m sorry sir @sampaio.diego, you are right.